package.json

```diff
@@ -26,7 +26,7 @@
     "less": "~2.6.0",
     "less-loader": "~2.2.3",
     "ng-cache-loader": "0.0.16",
-    "phantomjs-prebuilt": "~2.1.14",
+    "phantomjs-prebuilt": "2.1.7",
```

Comment:
phantomjs-prebuilt 2.1.7 still installs phantomjs 2.1.1, so there is not point in trying it.